### DISCUSSION TITLE
Download doesnt work

### DISCUSSION THREAD

**Discussion by CoolGAMERNL64** [brackets-cont/brackets#160] [Q&A] [answered]
When i try to download brackets using the "flatpak install flathub io.brackets.Brackets" command it tells me: 
~$ flatpak install flathub io.brackets.Brackets
Looking for matches…
error: Nothing matches io.brackets.Brackets in remote flathub
Can anyone help me fix this? Thanks in advance :)

**Comment by Pulkit-Pandey-09** [comment, marked as answer]
Duplication with #162 .  Closing this.
